manifests/a/AntibodySoftware/WizTree/4.13/AntibodySoftware.WizTree.installer.yaml

```
@@ -0,0 +1,27 @@
+# Automatically updated by the winget bot at 2023/Mar/16
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AntibodySoftware.WizTree
+PackageVersion: "4.13"
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: WizTree v4.12

```

```suggestion
- DisplayName: WizTree v4.13
```
@denelon - Can an engineer take a look at why this wasn't flagged for invalid Manifest-AppsAndFeaturesEntries?

---

Most if not all PRs I've seen that get flagged with this label are due to the DisplayVersion property being duplicated, the implementation doesn't check for DisplayName I believe.